website/src/word-panel.tsx

```diff
@@ -129,6 +126,28 @@ export const WordPanel = (p: {
   }
 }
 
+export const PartLines = (p: {
+  segments: Dailp.FormFieldsFragment["segments"]
+}) => {
+  if (p.segments) {
```

Comment:
It's usually clearer to make the exceptional case the early return. In this case, that means:

```ts
if (!p.segments) {
  return null
}
```

And then the normal function body after that.